guides/common/modules/proc_viewing-errata-by-applicable-and-installable.adoc

```diff
@@ -0,0 +1,13 @@
+[id="Viewing_Errata_by_Applicable_and_Installable_{context}"]
+= Viewing Errata by Applicable and Installable
+
+Use the following procedure to view errata by Applicable or Installable.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Select the host name.
+. Click the *Overview* tab. Under the Errata card, there are two types of Errata.
+. Click *Applicable* to view the type of Errata that applies to a package installed on your host.
+. Click *Installable* to view errata that are applicable errata available in the host’s content view and lifecycle environment.
```

Comment:
Aren't these supposed to be the opposite?

Reply:
Should it be reworded? @adamlazik1

Reply:
Could you please double check this with SME? To me, the description would make more sense if it was reversed. Could also be a misunderstanding on my part but still I will be happier if we can have this confirmed.

Reply:
@adamlazik1 Spoke to the SME and he said what I have is correct. Installable is a subset of Applicable.

Reply:
:+1: